include/unifex/scheduler_concepts.hpp

```diff
@@ -180,22 +190,22 @@ struct _schedule::sender {
 
   static constexpr bool sends_done = true;
 
-  template(
-    typename Receiver,
-    typename Scheduler =
-      std::decay_t<callable_result_t<decltype(get_scheduler), const Receiver&>>,
-    typename ScheduleSender = callable_result_t<decltype(schedule), Scheduler&>)
```

Comment:
Removing this line was the key change needed to fix msvc in my experiments.
Not sure why it is particularly problematic...

Reply:
Yeah, I don't know why. There was nothing wrong with the original code. MSVC doesn't make sense sometimes.